applications/concept-catalogue/src/main/kotlin/no/begrepskatalog/conceptregistration/HarvestEndpoint.kt

```diff
@@ -52,45 +54,62 @@ class HarvestEndpoint(val sqlStore: SqlStore) : CollectionsApi {
-                    .build()
-
-        begrep.bruksområde.forEach { bruksområde -> builder = builder.domainOfUse(bruksområde, "nb") }
-
-        var altLabelBuilder = builder.altLabelBuilder()
+
+        var conceptBuilder = collectionBuilder.conceptBuilder(urlForAccessingThisBegrepsRegistration)
+        val definitionBuilder = conceptBuilder.definitionBuilder()
+        val sourceDescriptionBuilder = definitionBuilder.sourcedescriptionBuilder()
+        val sourceBuilder = sourceDescriptionBuilder.sourceBuilder()
+
+
+        when (begrep.kildebeskrivelse.forholdTilKilde) {
+            Kildebeskrivelse.ForholdTilKildeEnum.EGENDEFINERT -> sourceDescriptionBuilder.sourcetype(SourceType.Source.Userdefined)
+            Kildebeskrivelse.ForholdTilKildeEnum.BASERTPAAKILDE -> sourceDescriptionBuilder.sourcetype(SourceType.Source.BasedOn)
+            Kildebeskrivelse.ForholdTilKildeEnum.SITATFRAKILDE-> sourceDescriptionBuilder.sourcetype(SourceType.Source.QuoteFrom)
+        }
+
+        begrep.kildebeskrivelse.kilde.forEach{
+            sourceBuilder.label(it.tekst,"nb")
+                    .seeAlso(it.uri)
+        }
+        sourceBuilder.build()
+        sourceDescriptionBuilder.build()
+
+        definitionBuilder
+            .text(begrep.definisjon, "nb")
+            .audience(AudienceType.Audience.Public)
+            .scopeNote(begrep.merknad ?: "", "nb")
+            .scopeBuilder()
+                .label(begrep.omfang?.tekst ?: "", "nb")
+                .seeAlso(begrep.omfang?.uri)
+            .build()
+            .modified(begrep.gyldigFom)
+        .build()
+
+        conceptBuilder
+            .identifier(begrep.id)
+            .publisher(begrep.ansvarligVirksomhet.id)
+            .prefLabelBuilder()
+                .label(begrep.anbefaltTerm ?: "", "no")
+            .build()
+            .example(begrep.eksempel, "nb")
+            .subject(begrep.fagområde, "nb")
+            .contactPointBuilder()
+                .email(begrep.kontaktpunkt?.harEpost ?: "")
+                .telephone(begrep.kontaktpunkt?.harTelefon ?: "")
+            .build()
+
+
+        begrep.bruksområde.forEach { bruksområde -> conceptBuilder = conceptBuilder.domainOfUse(bruksområde, "nb") }
+
+        var altLabelBuilder = conceptBuilder.altLabelBuilder()
         begrep.tillattTerm.forEach { tillattTerm -> altLabelBuilder = altLabelBuilder.label(tillattTerm, "nb") }
-        builder = altLabelBuilder.build()
+        conceptBuilder = altLabelBuilder.build()
 
-        var hiddenLabelBuilder = builder.hiddenLabelBuilder()
+        var hiddenLabelBuilder = conceptBuilder.hiddenLabelBuilder()
         begrep.frarådetTerm.forEach { frarådetTerm -> hiddenLabelBuilder = hiddenLabelBuilder.label(frarådetTerm, "nb") }
-        builder = hiddenLabelBuilder.build()
+        conceptBuilder = hiddenLabelBuilder.build()
 
-        return builder.resource
+        conceptBuilder.build()
+        return conceptBuilder.resource
```

Comment:
Is it possible to combine the two lines into:

return conceptBuilder.build().resource ?

Reply:
done